user-tools/README-CV Object Detection.md

```diff
@@ -0,0 +1,150 @@
+
+**TODO:** [Loom Video - Feel free to ignore this as I will be making these from my own demos]
+
+#### **CVObjectDetection via textQuery**
+
+When a textQuery is specified the AIService will search for the specified object in the image. For example, if the textQuery is "Dog" then the AIService will look for a dog in the image. If a dog is detected then the AIService will return the bounding box of the dog.
+
+This approach is useful when you want to detect an object that can look differently in the game. For example, you might want to click on a "Tree" but there might be several types of trees in the game, and you do not want to differentiate between them. If you know exactly what the tree might look like (such as a static sprite) then [CV Image](user-tools\README-CVImage.md) might be a better option for you.
+
+
+#### **CVObjectDetection via imageQuery**
+
+When an imageQuery is specified the AIService will search for the type of object that is contained in the image. For example, if the imageQuery is an image of a dog then the AIService will first identify that the main object in the imageQuery is a dog, then it will look for a dog in the image. If a dog is detected then the AIService will return the bounding box of the dog.
+
+This approach is useful when you have an image of an asset, however the asset can look different in the game. For example, the asset might be rotated, have an effect on it, or the rendering details might be different.
+
+## Configuration
+
+### Key JSON fields
+
+#### `endCriteria`
+- `type` - `CVObjectDetection` - tells the SDK to evaluate this type of criteria using the supplied data
+- `transient` - `transient`=`true` means that this image can match at any time during the evaluation of this bot segment and that passing result will persist even if it takes multiple more frames before other criteria in this segment are matched.  `transient`=`false` means that this criteria and other non-transient criteria must all be true at the same time (any transient criteria must also have matched already).  **Transient should almost always be `true` for `CVObjectDetection`**.  `CVObjectDetection` evaluation is a largely asynchronous process that can take multiple frames to complete, thus setting `trasient`=`false` for the criteria to match within a single frame can lead to situations where the criteria matched on the frame when the request was made, but are no longer matching by the time the result comes back.  Setting `transient`=`true` allows the needed flexibility for this asynchronous operation to pass more easily, especially when used  combination with other `endCriteria`.
+- `data` - The data json object that defines how to evaluate this `CVObjectDetection` criteria. Either imageQuery or textQuery can be specified, you can not use both.
```

Comment:
Since there was a lot of discussion around this, maybe we should make this bold.
```suggestion
- `data` - The data json object that defines how to evaluate this `CVObjectDetection` criteria. **Either imageQuery or textQuery can be specified, you can not use both.**
```